lib/progressTracker.js

```diff
@@ -34,7 +35,6 @@ function track (instance, opts) {
   let durationProgressBar
   let amountProgressBar
   let addedListeners = false
-  let spinner
```

Comment:
even if I move the instantiation here, it will still cause the test failure. moving it outside this method is what fixes it